Description: templates: mount devtmpfs in ubuntu containers
 That way /dev/disk/ exists.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/grub2/+bug/1060404
Forwarded: yes

Index: lxc/templates/lxc-ubuntu-cloud.in
===================================================================
--- lxc.orig/templates/lxc-ubuntu-cloud.in	2012-10-04 11:59:00.784184000 -0500
+++ lxc/templates/lxc-ubuntu-cloud.in	2012-10-04 12:02:27.983278932 -0500
@@ -94,6 +94,7 @@
     cat <<EOF > $path/fstab
 proc            proc         proc    nodev,noexec,nosuid 0 0
 sysfs           sys          sysfs defaults  0 0
+devtmpfs        dev          devtmpfs defaults 0 0
 EOF
 
     # rmdir /dev/shm for containers that have /run/shm
Index: lxc/templates/lxc-ubuntu.in
===================================================================
--- lxc.orig/templates/lxc-ubuntu.in	2012-10-04 11:59:00.784184000 -0500
+++ lxc/templates/lxc-ubuntu.in	2012-10-04 12:02:21.439278789 -0500
@@ -353,6 +353,7 @@
     cat <<EOF > $path/fstab
 proc            proc         proc    nodev,noexec,nosuid 0 0
 sysfs           sys          sysfs defaults  0 0
+devtmpfs        dev          devtmpfs defaults 0 0
 EOF
 
     if [ $? -ne 0 ]; then
